ledger/participant-state/kvutils/app/BUILD.bazel

```diff
@@ -64,6 +64,7 @@ da_scala_test_suite(
     srcs = glob(["src/test/**/*.scala"]),
     deps = [
         ":app",
+        "//daml-lf/data",
```

Comment:
Why is this dependency needed?

Reply:
Yes, otherwise you get
```
ledger/participant-state/kvutils/app/src/test/scala/com/daml/ledger/participant/state/kvutils/app/ConfigSpec.scala:14: error: value assertFromString is not a member of <notype>
  private val participantId: ParticipantId = ParticipantId.assertFromString("dummy-participant")
```

Reply:
Right. thank you.